docs/05-technical-reference/05-configuration-parameters/smsh-01-istio-chart.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Istio chart
+---
+
+To configure the Istio chart, override the default values of its [`values.yaml`](https://github.com/kyma-project/kyma/blob/main/resources/istio-configuratio/nvalues.yaml) file. This document describes parameters that you can configure.
```

Comment:
Wrong link (404).

Reply:
I updated and changed to [`values.yaml`](https://github.com/kyma-project/kyma/blob/main/resources/istio-configuration/values.yaml); however, the link will work only once Karol's PR is merged (the new chart is called `istio-configuration` and not `istio` as it is in Kyma 1.0.